### ISSUE TITLE
LoginButtons Dropdown broken

### ISSUE THREAD

**Issue by derouck** [andylash/meteor-accounts-admin-ui-bootstrap-3#2]
I've tried to use you package instead of the version of hharnisc but it seems the loginButtons dropdown is broken when I try to use your package.

Is this a known issue?

**Comment by andylash** [comment]
Not knowingly. I'll be happy to fix when I get back from vacation in a
week. Sorry to bother!

**Comment by andylash** [comment]
Pushed an update moving to the official bootstrap package, does that fix your problem?

**Comment by derouck** [comment]
Yes! The issue is indeed solved by removing the inofficial package.

Thanks
